wrapping/python/openmeeg/openmeeg.i

```diff
@@ -541,17 +575,21 @@ namespace OpenMEEG {
         }
 
         //  Create meshes and add triangles.
-
         for (unsigned i=0; i<N; ++i) {
             PyObject* item = PyList_GetItem(pylist,i);
             PyObject* name = PyList_GetItem(item,0);
             if (name==nullptr || !PyUnicode_Check(name))
                 throw Error(SWIG_TypeError, "Geometry constructor list of lists must each have first entry a non-empty string.");
-            Mesh& mesh = geometry->add_mesh();
+            Mesh& mesh = geometry->add_mesh(PyUnicode_AsUTF8(name));
             PyObject* triangles = PyList_GetItem(item,2);
             mesh_add_triangles(&mesh,triangles,indmap[i]);
             mesh.update(true);
+            std::ostringstream oss;
+            oss << "SWIG Geometry update of meshes(" << i << ")=\"" << mesh.name() << "\"";
+            for (const auto& mesh : geometry->meshes())
+                mesh.check_consistency(oss.str());
```

Comment:
Okay finally some progress I think -- there are now *two* failures using the `check_consistency` function, which is hopefully temporary but just iterates over all edges to try to recreate the indexing problem. It does so, as expected only on Windows, but it seems to happen earlier than I expected, here during creation of the *second* mesh (Skull) it messes up the Cortex mesh, which was previously consistent (well before even computing S or N):

```
Create Geometry from list of lists
    geom_add_vertices(geom, pyobj) done.
    geom_add_vertices(geom, pyobj) done.
    geom_add_vertices(geom, pyobj) done.
    mesh_add_triangles for mesh Cortex : reference_vertices : triangles : check done.
Mesh Cortex (00000143F55713D0) consistent during SWIG Geometry update of meshes(0)="Cortex"
    mesh_add_triangles for mesh Skull : reference_vertices : triangles : check done.

E   RuntimeError: Unknown vertex in triangle: Mesh Cortex invalid    during SWIG Geometry update of meshes(1)="Skull", requested triangle vertex address:
E     00000143F5589900 (00000143F7A56420)
E   but valid triangle vertex addresses are:
E     00000143F7A56960
E     00000143F7A56AE0
E     00000143F7A56B20
```
or even worse/weirder in the other failure:
```
E   RuntimeError: Unknown vertex in triangle: Mesh Cortex invalid    during SWIG Geometry update of meshes(1)="Skull", requested triangle vertex address:
E     00000143F4B242A0 (00000143F557CEC0)
E   but valid triangle vertex addresses are:
E     00000143F557D5C0
E     0000000000000000
E     0000000000000000
```
https://github.com/openmeeg/openmeeg/runs/8181109365?check_suite_focus=true

I'm going to stop now, @papadop feel free to take over if you have some ideas here!